### PR TITLE
Allow disabling guest login

### PR DESCRIPTION
**Pull request by JFronny**

### CODE REVIEW

File: src/main/java/edu/kit/hci/soli/domain/SystemConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import lombok.ToString;
+
+/**
+ * Entity class representing the system configuration.
+ * This table is designed to contain only a single row.
+ */
+@Entity
+@Table(name = "soli_system_configuration")
+@ToString
+public class SystemConfiguration {
```
Comment (Jan8192) [inline src/main/java/edu/kit/hci/soli/domain/SystemConfiguration.java:15]: I'm not sure if this is correct approach to storing application settings... But what other alternatives do we have? I guess this is fine

File: src/main/jte/deactivate_user_confirmation.jte [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+@param User user
+
+@template.layout(page = new PageSpec(context.lookup("generic.warning"), "Soli"), context = context, login = login, room = null, content = @`
+    <div class="card bg-base-200 w-80 mx-auto">
+        <div class="card-body">
+            <h2 class="card-title">${context.localize("generic.warning")}</h2>
+            <p>${context.localize("disable.user.disable.are_you_sure")}</p>
+            <div class="modal-action">
+                <a class="btn" href="/admin/users">${context.localize("generic.cancel")}</a>
```
Comment (Jan8192) [inline src/main/jte/deactivate_user_confirmation.jte:20]: I dont why this happens but for some reason my browser redirects me to here: 
image
Do you have the same issue? Could this be related to this?

**Review by Jan8192 — CHANGES_REQUESTED** [review]

File: src/main/jte/disable_guests_confirmation.jte [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+@param CsrfToken csrf
+
+@template.layout(page = new PageSpec(context.lookup("administration.user.guests.disable"), "Soli"), context = context, login = login, room = null, content = @`
+    <div class="card bg-base-200 w-96 mx-auto">
+        <div class="card-body">
+            <h2 class="card-title">${context.localize("generic.warning")}</h2>
+            <p>${context.localize("administration.user.guests.disable.description")}</p>
+            <div class="modal-action">
+                <a class="btn" href="/admin/users">${context.localize("generic.cancel")}</a>
+                @template.form(csrf = csrf, cssClass = "card-actions justify-end", action = "//admin/users/disable-guests", method = PUT, content = @`
```
Comment (Jan8192) [inline src/main/jte/disable_guests_confirmation.jte:18]: I think I've spotted your bug, this should be /admin/users/disable-guests

**Review by Jan8192 — CHANGES_REQUESTED** [review]

File: src/main/java/edu/kit/hci/soli/domain/SystemConfiguration.java (hunk 1 above)
Comment (JFronny) [reply src/main/java/edu/kit/hci/soli/domain/SystemConfiguration.java:15]: I agree that this feels dirty, but according to StackOverflow, doing this should be just fine if we add a database constraint to maintain the singleton nature (which I have done)

**Review by Jan8192 — APPROVED** [review]
LGTM!

**Pull request merged**